spec/numberformat.html

```diff
@@ -409,12 +414,6 @@ <h1>Intl.NumberFormat.prototype.resolvedOptions ( )</h1>
           1. Let _v_ be the value of _nf_'s internal slot whose name is the Internal Slot value of the current row.
           1. If _v_ is not *undefined*, then
             1. Perform ! CreateDataPropertyOrThrow(_options_, _p_, _v_).
-        1. If _nf_.[[RoundingType]] is ~morePrecision~, then
-          1. Perform ! CreateDataPropertyOrThrow(_options_, *"roundingPriority"*, *"morePrecision"*).
-        1. Else if _nf_.[[RoundingType]] is ~lessPrecision~, then
-          1. Perform ! CreateDataPropertyOrThrow(_options_, *"roundingPriority"*, *"lessPrecision"*).
-        1. Else,
-          1. Perform ! CreateDataPropertyOrThrow(_options_, *"roundingPriority"*, *"auto"*).
```

Comment:
this seems normative (on the editorial commit) - does adding it to the table below cause it to be implicitly set on `options` instead?

Reply:
Yes.

There is a normative change, which is that the order in which the append order for the options is changing slightly: previously roundingPriority was at the end, but now it is second from the end.

Reply:
In the case, the commit is mislabeled; it'd help my review, at least, to split up the editorial and normative parts.

Reply:
The editorial and normative parts **are** separated, as explained in the PR description.
1. [`1ee3bee39e4b2bc21cb9105b7fba2c0c7cef4627`](https://github.com/tc39/ecma402/pull/768/commits/1ee3bee39e4b2bc21cb9105b7fba2c0c7cef4627) is editorial, moving the infallible computation of rounding priority from `NumberFormat.prototype.resolvedOptions` to the `Intl.NumberFormat` constructor and updating the [`resolvedOptions` algorithm](https://tc39.es/ecma402/#sec-intl.numberformat.prototype.resolvedoptions) to add a "roundingPriority" property from the table in step 5 rather than from special steps following table row iteration (but still adding "roundingPriority" as the observably last property).
2. [`0d936078aaf3d83b114ae092f4ac80e7bbfbcf85`](https://github.com/tc39/ecma402/pull/768/commits/0d936078aaf3d83b114ae092f4ac80e7bbfbcf85) is normative, reordering table rows such that "roundingPriority" will be grouped with the other "rounding…" properties and observably precede "trailingZeroDisplay".

Reply:
This comment is on the editorial commit.

Reply:
So what part of «**Editorial: Move computation of NF resolved option "roundingPriority" to initialization**» is mislabeled?

Reply:
@gibson042 that was my question. I only looked at the editorial commit, and I saw the line I'm commenting on here. @sffc said it's normative. Your comment implies that it's not normative ("but still"), which was what i'd been asking about.